examples/stable_diffusion_v2/ldm/modules/encoders/text_encoder.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, d_model, n_head, dtype=ms.float32):
         self.scaling = self.head_dim**-0.5
         self.dtype = dtype
 
-    def construct(self, query, key, value, attn_mask):
+    def construct(self, query, key=None, value=None, causal_attention_mask=None, attention_mask=None):
```

Comment:
better to keep the original arguments, inserted new arguments after them, prevent others model unbroken